tests/utils/test_env.py

```diff
@@ -972,12 +966,8 @@ def test_env_finds_the_correct_executables(tmp_dir, manager):
     default_pip_executable = expected_pip_executable = "pip" + (
```

Comment:
Use an f-string here too. I also wonder if there's a decent way to DRY this up -- maybe not in this PR.